index.d.ts

```diff
@@ -133,7 +133,8 @@ Parse a query string into an object. Leading `?` or `#` are ignored, so you can
 The returned object is created with [`Object.create(null)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create) and thus does not have a `prototype`.
 
 @param query - The query string to parse.
-*/
+@param options - The options describing how to parse the query string.
```

Comment:
This is moot. It doesn't say anything that's not already clear.

Reply:
It was just absent in the docs and highlighted by static code analyzer :)

Reply:
Agree, I'll revert it